index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable quote-props */
+/* eslint-disable no-use-before-define */
+import fetch from 'node-fetch';
+
+import xlsx from 'xlsx';
+
+import inquirer from 'inquirer';
+
+import dotenv from 'dotenv';
+
+// import SearchItem from './SearchItem';
+
+dotenv.config();
+class SearchItem {
+
+  constructor(title, description, link) {
+    this.title = title;
+    this.description = description;
+    this.link = link;
+  }
+
+}
+
+let searchName = (await inquirer.prompt({
+  name: 'search_name',
+  type: 'input',
+  message: 'What do you want to search in google?', 
+})).search_name;
+
+let searchNumber = (await inquirer.prompt({
+  name: 'search_number',
+  type: 'input',
+  message: 'How many search results do you want? (max 100)', 
+})).search_number;
```

Comment:
ideally we want validation that the user added something along that range. write a TODO to do that later